templates/ingress.yaml

```
@@ -0,0 +1,27 @@
+{{- if .Values.backend.ingress.enabled -}}
+apiVersion: extensions/v1beta1
```

ingress is at networking.k8s.io/v1 in 1.14+
```suggestion
apiVersion: networking.k8s.io/v1
```